lib/node_modules/@stdlib/utils/is-uppercase/lib/is_uppercase.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+// MODULES //
+
+var isString = require( '@stdlib/utils/is-string' ).isPrimitive;
+
+
+// MAIN //
+
+/**
+* Tests if a value is an uppercase string.
+*
+* @param {*} value - value to test
+* @returns {boolean} boolean indicating whether value is an uppercase string
+*
+* @example
+* var bool = isUppercase( 'HELLO' );
+* // returns true
+* @example
+* var bool = isUppercase( 'World' );
+* // returns false
+* @example
+* var bool = isUppercase( '!' );
+* // returns true
+* @example
+* var bool = isUppercase( 'salt and light' );
+* // returns false
+*/
+function isUppercase( value ) {
+	return isString( value ) && ( value.valueOf() === value.toUpperCase() );
```

Comment:
If `value` is a primitive, why use `valueOf()`? Can use `uppercase` string utility.